app/src/main/res/values/strings.xml

```diff
@@ -180,31 +180,31 @@
     <string name="sort_db">DB sort order</string>
     <string name="sort_title">Title</string>
     <string name="sort_username">Username</string>
-    <string name="sort_creation_time">Creation Time</string>
-    <string name="sort_last_modify_time">Last Modification Time</string>
-    <string name="sort_last_access_time">Last Access Time</string>
+    <string name="sort_creation_time">Created</string>
+    <string name="sort_last_modify_time">Changed</string>
+    <string name="sort_last_access_time">Accessed</string>
     <string name="special">Special</string>
     <string name="search">Search</string>
     <string name="search_results">Search results</string>
     <string name="underline">Underline</string>
     <string name="unsupported_db_version">Unsupported database version.</string>
     <string name="uppercase">Upper-case</string>
-    <string name="use_saf_summary">Use Android storage access (SAF) framework for file browsing (KitKat and later)</string>
+    <string name="use_saf_summary">Use Android storage access (SAF) framework for file browsing (KitKat onwards).</string>
     <string name="use_saf_title">Storage Access Framework</string>
     <string name="warning">Warning</string>
-    <string name="warning_password_encoding">Your password may contain letters not supported by the Latin-1 character set used by .kdb files. As these are all converted to the same letter, it is recommended to change your password to make it safer.</string>
-    <string name="warning_read_only">Your SD card currently read-only. You may not be able to save changes to your database.</string>
-    <string name="warning_unmounted">Your SD card not currently mounted on your device. You will not be able to load or create your database.</string>
+    <string name="warning_password_encoding">As all letters of your password not supported in Latin-1 are converted to the same letter, it is recommended to make it compliantly unique.</string>
```

Comment:
New version gets to the point quicker, with less complexion, and is shorter. I don't see how the meaning is changed.

Reply:
> it is recommended to make **it** compliantly unique.

 The "it" is supposed to reference the last name, here the letter and not the password which is confusing because we have to make the password unique and not just the letter.